api/src/main/java/io/strimzi/api/kafka/model/KafkaExporterSpec.java

```diff
@@ -77,6 +80,28 @@ public void setTopicRegex(String topicRegex) {
         this.topicRegex = topicRegex;
     }
 
+    @Description("Regular expression to specify which consumer groups to exclude. " +
```

Comment:
I think that there is simply no default value => so I would remove the `Default value is null.`.